doc/user/content/guides/dbt.md

```diff
@@ -0,0 +1,276 @@
+---
+title: "dbt and Materialize"
+description: "How to use dbt and Materialize to transform streaming data in (near) real-time."
```

Comment:
Nit: is there a reason why we're qualifying real time with `(near)`?

Reply:
Just force of habit, there's always someone who goes "well, actually". But it makes sense to drop it, I'll clean the guide up!